plugins/node.d/icinga

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+##%# family=auto
+
+#%# capabilities=autoconf
+
+ICINGACLI=${ICINGACLI:-$(which icingacli)}
+JQ=${JQ:-$(which jq)}
+
+
+if [ "$1" = "autoconf" ] ; then
+    if [ "$ICINGACLI" != ""  -a "$JQ" != "" ] ; then
+        echo yes
+        exit 0
+    else
+        echo "no (could not found icingacli or jq)"
+        exit 1
```

Comment:
"autoconf" should *always* exit with zero. A non-zero exitcode indicates a plugin failure (not a missing requirement).

Reply:
also changed.